### PR TITLE
Add Zyxel cfg lanhosts get

### PR DESCRIPTION
**Pull request by elavaud**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Zyxel OS, cfg lanhosts get

### CODE REVIEW

File: ntc_templates/templates/zyxel_os_cfg_lanhosts_get.textfsm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Value HOSTNAME (.+\S)
+Value IP (\d+\.\d+\.\d+\.\d+)
+Value IPV6 (.+?)
```
[the remark this answers is not in the document]
Comment (elavaud) [reply ntc_templates/templates/zyxel_os_cfg_lanhosts_get.textfsm]: Sorry I though a push the correction but no. Just added it

**Comment by jvanderaa** [comment]
Re-running the build.

**Review by jmcgill298 — COMMENTED** [review]
needs rebase

**Comment by elavaud** [comment]
Grouped in #1142

**Pull request closed without merging**